### PR TITLE
Improved SplitPackages

### PR DESCRIPTION
**Pull request by javiercn**
Addressed feedback from https://github.com/aspnet/DnxTools/pull/16

### CODE REVIEW

File: src/SplitPackages/Program.cs [hunk 1]
```diff
@@ -5,26 +5,86 @@
...
+                CommandOptionType.SingleValue);
+
+            var whatIfOption = app.Option(
+                "--whatif",
+                "Performs a dry run of the command with the given arguments without copying the packages",
+                CommandOptionType.NoValue);
+
+            var program = new Program(app, sourceOption, csvOption, destinationOption, whatIfOption);
+
+            app.OnExecute(new Func<int>(program.Execute));
```
Comment (Eilon) [inline src/SplitPackages/Program.cs]: Wondering - can you just pass in the delegate and let the compiler do the work?

```c#
app.OnExecute(program.Execute);
```

**Comment by Eilon** [comment]
Looks good :shipit:

File: src/SplitPackages/Program.cs (hunk 1 above)
Comment (javiercn) [inline src/SplitPackages/Program.cs]: The compiler is not smart enough to determine the right overload to use, so I need to rely on the new Func<...>

Comment (Eilon) [inline src/SplitPackages/Program.cs]: Ah ok no worries.

**Pull request merged**